README.md

```diff
@@ -45,6 +45,28 @@ $ swift package generate-xcodeproj --xcconfig-overrides overrides.xcconfig
 $ open SourceKitLSP.xcodeproj
 ```
 
+## Debugging
+
+You can attach LLDB to SourceKit-LSP and set breakpoints to debug. You may want to instruct LLDB to wait for the sourcekit-lsp process to launch and then start your editor, which will typically launch
+SourceKit-LSP as soon as you open a Swift file:
+
+
+```sh
+(lldb) process attach --name sourcekit-lsp --waitfor
```

Comment:
The way I always do this is to run lldb 

```
$ lldb -w -n sourcekit-lsp
```

The way you documented it is fine with me, just thought I'd mention it.